src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
+      <li>
+        <b>{post.title}</b>
```

Comment:
It's better to replace tag `b` with `strong` because  strong is semantic tag
```suggestion
        <strong>{post.title}</strong>
```